config/routes.rb

```diff
@@ -247,4 +239,18 @@
 
   # Startpage
   root :to => 'home#show'
+
+  #OpenID Connect & OAuth
+  namespace :openid_connect do
+    resources :clients, only: :create
+    post 'access_tokens', to: proc { |env| OpenidConnect::TokenEndpoint.new.call(env) }
```

Comment:
Space between { and | detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.